Build/Build.cs

```diff
@@ -115,7 +115,7 @@ class Build : NukeBuild
         {
             DotNetBuild(s => s
                 .SetProjectFile(Solution)
-                .SetConfiguration("CI")
```

Comment:
Just for interest: why is this configuration `CI` needed?

Reply:
To reduce what projects we build during CI.
E.g. no need to build benchmarks projects as we don't run it during CI.

Reply:
Ahh.. I missed this specific projects :)
thanks

Reply:
It's easiest to get an overview of, if you look at it in your IDE's configuration manager.
In the CI configuration we also:
* Compile the library in release mode as that's the dll that's included in the nupkg, but 
* Compile the tests in debug mode as that's needed for CallerIdentification

Reply:
Quite cool :)